cms/templates/js/video-thumbnail.underscore

```diff
@@ -9,7 +9,10 @@
         </label>
 
         <span class="requirements-text-sr sr">
-            <%- gettext('Recommended image resolution is 1280x720 pixels and format must be one of .jpg, .png or .gif') %>
+            <%- edx.StringUtils.interpolate(
+                gettext("Recommended image resolution is 1280x720 pixels, maximum image file size should be {maxFileSize} and format must be one of {supportedImageFormats}."),
```

Comment:
This is only for screen reader.

<img width="683" alt="screen shot 2017-06-01 at 4 27 30 pm" src="https://cloud.githubusercontent.com/assets/6991154/26677916/6a6e60a0-46e7-11e7-848d-5529b9a36b81.png">